test/features/cloudnative/network_zones/network_zones.go

```diff
@@ -9,11 +9,11 @@ import (
 	"time"
 
 	"github.com/Dynatrace/dynatrace-operator/pkg/api/status"
-	dynakubev1beta2 "github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta2/dynakube"
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta2/dynakube"
 	"github.com/Dynatrace/dynatrace-operator/test/features/cloudnative"
 	"github.com/Dynatrace/dynatrace-operator/test/helpers"
 	"github.com/Dynatrace/dynatrace-operator/test/helpers/components/activegate"
-	"github.com/Dynatrace/dynatrace-operator/test/helpers/components/dynakube"
+	dynakubeComponents "github.com/Dynatrace/dynatrace-operator/test/helpers/components/dynakube"
```

Comment:
```suggestion
	dynakubeHelpers "github.com/Dynatrace/dynatrace-operator/test/helpers/components/dynakube"
```

Reply:
I like my version because its a project folder name rather than a top-level folder. But I prefer to holy war it in separate PR.